assets/scripts/background.js

```diff
@@ -20,6 +20,10 @@ async function createOffscreen() {
 
 chrome.runtime.onStartup.addListener(createOffscreen);
 chrome.runtime.onInstalled.addListener(createOffscreen);
+chrome.runtime.onConnect.addListener(function(port) {
```

Comment:
Just to follow the design pattern we can change this statement to an arrow function.

```js
chrome.runtime.onConnect.addListener(port =>{
  chrome.offscreen.closeDocument();
  createOffscreen();
});
```